checker/main.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+
+	"github.com/systemboot/systemboot/pkg/checker"
+)
+
+var configFile string
+
+func init(){
+	flag.StringVar(&configFile, "config-file", "", "Config file to use")
+}
+
+func usage(){
+	fmt.Fprintf(os.Stderr, "Usage of %s:\n", os.Args[0])
+	flag.PrintDefaults()
+}
+
+func run() int {
+	if configFile == "" {
+		fmt.Fprintf(os.Stderr, "Error: config-file argument is required\n")
+		usage()
+		return 1
+	}
+
+	log.Printf("Registered Checks: %v\n", checker.ListRegistered())
+
+	var checklist []checker.Check
+
+	checkerConfigStr, err := ioutil.ReadFile(configFile)
+	if err != nil {
+		log.Printf("Unable to open config file %#v: %s", configFile, err.Error())
```

Comment:
you should invert the format strings - use `%s` for `configFile` and `%v` for `err`. Also just use `err`:
```suggestion
		log.Printf("Unable to open config file %s: %v", configFile, err)
```
(note: when moving `configFile` outside `init`, you will need to dereference it here)